packages/web/docs/next.config.js

```diff
@@ -147,6 +147,11 @@ export default withGuildDocs({
       destination: '/docs/other-integrations/:path*',
       permanent: false,
     },
+    {
+      source: '/docs/api-reference/gateway/cli',
+      destination: 'docs/gateway',
```

Comment:
this redirect destination seems incorrect. https://the-guild.dev/graphql/hive/docs/api-reference/gateway-cli seems to be the better fit.